### PR TITLE
Drop _reference from reference file names

### LINKED ISSUE #141: Remove the trailing `_reference` from reference result files
The trailing `_reference` at the end of the filename of reference result files does not really serve a purpose. Let's remove it.

### PR DESCRIPTION
**Pull request by davidrudlstorfer**
Closes #141

### CODE REVIEW

File: tests/testing_meshpy.py [hunk 1]
```diff
@@ -300,7 +300,9 @@ def create_comments_in_solid(self, full_import):
         # Convert the solid mesh to meshpy objects.
         mpy.import_mesh_full = full_import
 
-        solid_file = os.path.join(testing_input, "test_meshpy_comments_in_solid.dat")
+        solid_file = os.path.join(
+            testing_input, "test_meshpy_comments_in_solid_initial.dat"
```
Comment (isteinbrecher) [inline tests/testing_meshpy.py:304]: This is changed, because otherwise the "new" reference file would have the same name, right?

**Review by isteinbrecher — APPROVED** [review]
Thanks a lot @davidrudlstorfer, looks good to me!

File: tests/testing_meshpy.py (hunk 1 above)
Comment (davidrudlstorfer) [reply tests/testing_meshpy.py:304]: Yes exactly - I actually thought too long about the name - we could also take something like `base` or `original`. If you'r fine with `initial` I would merge

Comment (isteinbrecher) [reply tests/testing_meshpy.py:304]: `initial` is fine with me. I just wanted to double check. Let's merge this!

**Pull request merged**